ts/src/pro/poloniex.ts

```diff
@@ -0,0 +1,965 @@
+//  ---------------------------------------------------------------------------
+
+import poloniexRest from '../poloniex.js';
+import { BadRequest, AuthenticationError } from '../base/errors.js';
+import { ArrayCache, ArrayCacheByTimestamp, ArrayCacheBySymbolById } from '../base/ws/Cache.js';
+import { Precise } from '../base/Precise.js';
+import { sha256 } from '../static_dependencies/noble-hashes/sha256.js';
+
+//  ---------------------------------------------------------------------------
+
+// @ts-expect-error
+export default class poloniex extends poloniexRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchOHLCV': true,
+                'watchOrderBook': true,
+                'watchTicker': true,
+                'watchTickers': true,
+                'watchTrades': true,
+                'watchBalance': true,
+                'watchStatus': false,
+                'watchOrders': true,
+                'watchMyTrades': false,
```

Comment:
@samgermain I think we can implement watchMyTrades with the order's `stream` 
![image](https://user-images.githubusercontent.com/43336371/227914679-228d2b99-e53e-4e5d-af72-24856e36e274.png)

Reply:
It turns out I that I didn't save the changes when I wrote `watchMyTrades` but you should just be able to pass `"eventType": "trade"` via `params`